DEVELOPERS.md

```diff
@@ -523,3 +526,61 @@ inversion of control so the calling view can decide the context in which
 presenters are used.
 This will most likely affect the template used for each event, and where each
 object links to, if anywhere.
+
+## Interfaces
+
+Descriptions of interfaces between this repo or container and others. These
+interfaces can be changed, through coordination with the relevant teams.
+
+Except where mentioned otherwise, URLs are relative to the root of the
+job-server API endpoint (https://jobs.opensafely.org/api/v2 in production).
+
+### [job-runner]
+
+[job-runner] is the container that runs in a secure backend and executes
+[JobRequest]s initiated by users of job-server.
+
+This interacts with [jobserver/api/jobs.py] It uses the `JobRequestAPIList`
+endpoint (`GET /job-requests/`) for reading `JobRequests`. It uses the
```

Comment:
Nitpick: `JobRequest`s

Reply:
Applied, and `Job` table rather than `Jobs` table, thanks